### PR TITLE
Side bar Animate

### PR DESCRIPTION
**Pull request by aashreya**

### CODE REVIEW

**Comment by aashreya** [comment]
@tommysullivan Need some help with this. The current implementation broke with the castle risk page, need some help in sending the classes from parent components to children.

**Comment by tommysullivan** [comment]
Hey! If you got it working pre rebase then I’m sure we can get it! We can pair

File: src/components/welcome/HomePage.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import * as React from "react";
+import classNames from 'classnames';
 
-export const HomePage = () => <div className="jumbotron vertical-center">
+export const HomePage = (props) => {
+    const {classes, isSideNavOpen} =props;
```
Comment (tommysullivan) [inline src/components/welcome/HomePage.tsx]: Can we kill these props?

Comment (aashreya) [reply src/components/welcome/HomePage.tsx]: Ah yes will delete them before merging.

**Pull request merged**